### DISCUSSION TITLE
Feature Request: Rules to block certain message in issue

### DISCUSSION THREAD

**Discussion by Sam-Lin-MillersLab** [community/community#78033] [Projects and Issues]
### Select Topic Area

Product Feedback

### Feature Area

Issues

### Body

In any popular issue, there are always some folks like to post +1 , same here. It is really a waste of resources and annoy to whoever subscribes to it. I think if there is a rule to block this kind of message will be awesome and probably can save some resource for Github as well.

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by tnrich** [comment]
+1
